.pre-commit-config.yaml

```diff
@@ -26,6 +26,7 @@ repos:
     -   id: autopep8
         args:
         - -i
+        - -v
```

Comment:
why adding the verbose here ?

Reply:
Sometimes autopep8 modifies files during PR# pre-commit checks (even if a local pre-commit run finished without any modifications) and it's hard to understand which files are modified.

Reply:
o.k.